corehq/apps/hqmedia/tasks.py

```diff
@@ -133,7 +133,7 @@ def process_bulk_upload_zip(processing_id, domain, app_id, username=None, share_
 
 
 @task(serializer='pickle')
-def build_application_zip(include_multimedia_files, include_index_files, app,
+def build_application_zip(include_multimedia_files, include_index_files, domain, app_id,
```

Comment:
similar to https://github.com/dimagi/commcare-hq/pull/23398#discussion_r263021916 I believe this will cause any builds in the queue to fail after a deploy